core/src/main/java/com/novoda/accessibility/ClosedCaptionManagerFactory.java

```diff
@@ -0,0 +1,63 @@
+package com.novoda.accessibility;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.os.Build;
+import android.view.accessibility.CaptioningManager;
+
+public class ClosedCaptionManagerFactory {
+
+    public static CaptionManager createCaptionManager(Context context) {
```

Comment:
to the from. `from` _only only only_ makes sense if **all** of the following hold:

- it's a static method
- the class that the static method is in is the type that you're converting _to_
- the single parameter you pass to the static method is the type that you're converting _from_